presto-native-execution/testcontainers/nativeworker/velox-etc/velox.properties

```diff
@@ -0,0 +1 @@
+mutable-config=true
```

Comment:
Is this required--if not, can we remove it?

Reply:
Not need for now. Removed